Packs/NistNVD/Integrations/NistNVD/NistNVD.py

```diff
@@ -0,0 +1,322 @@
+# import datetime
+import sys
+from datetime import datetime, timedelta
+
+import demistomock as demisto  # noqa: F401
+import requests
+from CommonServerPython import *  # noqa: F401
+# from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+# !/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# disable insecure warnings
+# requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+''' CONSTANTS '''
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+VERIFY_SSL = not demisto.params().get('insecure', False)
+
+if not demisto.params().get('proxy', False):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+
+def test_module() -> str:
+    try:
+        base_url = urljoin(demisto.params()['url'], '/rest/json/cves/1.0')
+        headers = {'Accept': 'application/json'}
+        req = requests.get(base_url, headers=headers, verify=VERIFY_SSL)
+        if req.status_code != 200:
+            return_results(req.status_code)
+            sys.exit(1)
+    except Exception as e:
+        if ('Forbidden' in str(e)):
+            return_results(req.status_code)
```

Comment:
If you have more descriptive error messages it would really help the users.
And another thing, the `return_error` method is encapsulating both the war-room print and the `sys.exit` together so you can change it to use it.

Reply:
Thanks @teizenman, You are right and I used return_error.